films.JSON

```diff
@@ -48,5 +48,18 @@
         "EmptySeats":50,
         "url": "https://img.afisha.tut.by/static/media/340x0s/cover/09/8/zelenaya-kniga-6781953.jpg"
```

Comment:
why using urls in sessions, they are already defined on films

Reply:
i will do it in the next PR